examples/ScrollToColumnExample.js

```diff
@@ -115,20 +112,8 @@ class ScrollToColumnExample extends React.Component {
           headerHeight={50}
           width={1000}
           height={500}
-          scrollableColumnsCount={this.state.scrollableColumnsCount}
-          getScrollableColumn={(i) =>
-            this.state.columns[this.state.fixedColumnsCount + i]
-          }
-          getFixedColumn={(i) => this.state.columns[i]}
-          fixedColumnsCount={this.state.fixedColumnsCount}
-          fixedRightColumnsCount={this.state.fixedRightColumnsCount}
-          getFixedRightColumn={(i) =>
-            this.state.columns[
-              this.state.fixedColumnsCount +
-                this.state.scrollableColumnsCount +
-                i
-            ]
-          }
+          columnsCount={this.state.columnsCount}
```

Comment:
Can this just be `state.colums.length`?  This will let us remove `state.columnsCount`